packages/backend/src/controllers/v2/ChartsController.ts

```diff
@@ -0,0 +1,61 @@
+import {
+    ApiChartContentResponse,
+    ApiErrorPayload,
+    ChartContent,
+    ContentType,
+    KnexPaginatedData,
+} from '@lightdash/common';
+import {
+    Get,
+    Hidden,
+    Middlewares,
+    OperationId,
+    Query,
+    Request,
+    Response,
+    Route,
+    SuccessResponse,
+    Tags,
+} from '@tsoa/runtime';
+import express from 'express';
+import { allowApiKeyAuthentication, isAuthenticated } from '../authentication';
+import { BaseController } from '../baseController';
+
+@Route('/api/v2/charts')
+@Response<ApiErrorPayload>('default', 'Error')
+@Tags('v2', 'Charts')
+@Hidden() // Hide this endpoint from the documentation for now
+export class ChartsController extends BaseController {
+    /**
+     * Get charts
+     */
+    @Middlewares([allowApiKeyAuthentication, isAuthenticated])
+    @SuccessResponse('200', 'Success')
+    @Get('/')
+    @OperationId('List charts')
+    async listCharts(
+        @Request() req: express.Request,
+        @Query() projectUuids?: string[],
+        @Query() spaceUuids?: string[],
```

Comment:
> I assume we don't want to support the comma separated list like ?spaceUuids=23f4893b-0707-4a0d-8ab9-7d3f7003b355,09fcfcb1-8d61-4383-b1c5-63a9034b3824

@javi correct. The correct way to support multiple values (array) is to pass the same param multiple times
```
# get charts from multiple spaces
/api/v2/charts?spaceUuids=d62ffb01-0463-4180-b8de-45b6ca5c34cb&spaceUuids=1539ce96-58d4-483b-b29e-16df926704d6
```